app/main/views/buyers.py

```diff
@@ -158,6 +158,11 @@ def view_brief_overview(framework_slug, lot_slug, brief_id):
     if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id):
         abort(404)
 
```

Comment:
Ternary assignment can fit this on 2 lines :man_shrugging: 
Total personal style thing. Up to you.